sirepo/pkcli/flash.py

```diff
@@ -35,9 +35,10 @@ def _get_file(dest):
         _remote_file(dest)
 
     def _local_file(dest):
-        shutil.copy(pkio.py_path(
-            cfg.dev_depot_url.replace(_FILE_PREFIX, ''),
-        ).join(dest.basename), dest)
+        p = pkio.py_path(cfg.dev_depot_url.replace(_FILE_PREFIX, ''))
```

Comment:
Since dev_depot_url is only used by the setup_dev command, make it an arg to the function with a default value.

Reply:
https://github.com/radiasoft/sirepo/pull/2490/commits/00903ca4af2b437c4de426a53cfab1f8b79638eb